src/elm/Page/Login.elm

```diff
@@ -132,3 +141,5 @@ msgToString msg =
     case msg of
         GotAuthMsg subMsg ->
             "GotAuthMsg" :: Auth.msgToString subMsg
+
+        _ -> []
```

Comment:
In order to avoid stuff like this, please configure your text editor to add a newline at the end of the file when it saves, its a feature every editor should have an option by default (without extensions)